fastmigrate/core.py

```diff
@@ -489,21 +487,21 @@ def run_migrations(
             # Update version
             _set_db_version(db_path, version)
             if verbose:
-                console.print(f"[green]✓[/green] Database updated to version [bold]{version}[/bold] [dim]({migration_duration:.2f}s)[/dim]")
+                print(f"✓ Database updated to version {version} ({migration_duration:.2f}s)")
         
         # Show summary of successful run
         if stats["applied"] > 0 and verbose:
             total_duration = time.time() - start_time
-            console.print("\n[bold green]Migration Complete[/bold green]")
-            console.print(f"  • [bold]{stats['applied']}[/bold] migrations applied")
-            console.print(f"  • Database now at version [bold]{sorted_versions[-1]}[/bold]")
-            console.print(f"  • Total time: [bold]{total_duration:.2f}[/bold] seconds")
+            print("\nMigration Complete")
+            print(f"  • {stats['applied']} migrations applied")
+            print(f"  • Database now at version {sorted_versions[-1]}")
+            print(f"  • Total time: {total_duration:.2f} seconds")
```

Comment:
BTW we can remove the calculation of `total_duration` -- they can always use unix `time` cmd if they want.

Reply:
Resolved via a42505d